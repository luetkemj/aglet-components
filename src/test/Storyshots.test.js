import initStoryshots from '@storybook/addon-storyshots';

const addonInfo = require('@storybook/addon-info');

addonInfo.withInfo = () => storyFn => storyFn;

initStoryshots();
