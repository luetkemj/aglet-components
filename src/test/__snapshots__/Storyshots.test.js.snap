// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Footer basic 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <div
    className="container"
  >
    <div
      className="actions"
    >
      <div
        className="left"
      >
        <p>
          <a
            href="http://luetkemj.github.io/"
          >
            Made with 
            <span
              className="heart"
            >
              â™¥
            </span>
             in Austin, Texas
          </a>
        </p>
      </div>
      <div
        className="right"
      >
        <p
          className="bug"
        >
          <a
            href="http://github.comissues"
          >
            Report a bug
          </a>
        </p>
        <p
          className="feature"
        >
          <a
            href="http://github.comissues"
          >
            Request a feature
          </a>
        </p>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Header basic 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <div
    className="container"
  >
    <div
      className="masthead"
    >
      <div
        className="logo"
      >
        <img
          alt="logo"
          src="https://s3-us-west-2.amazonaws.com/agletio/images/nib.svg"
        />
      </div>
      <div
        className="nameplate"
      >
        <div
          className="name"
        >
          AGLET
        </div>
        <div
          className="description"
        >
          Tools for the discerning Game Master
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots InputNumber Basic 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputNumber"
  >
    <p
      className="label"
    >
       
    </p>
    <input
      className="input"
      name="inputNumber"
      onChange={[Function]}
      placeholder={null}
      type="number"
      value="0"
    />
  </label>
</div>
`;

exports[`Storyshots InputNumber Controlled InputNumber 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputNumber"
  >
    <p
      className="label"
    >
      Number Input Field
       
      <span
        className="required"
      >
        *
      </span>
    </p>
    <input
      className="input"
      name="inputNumber"
      onChange={[Function]}
      placeholder={null}
      type="number"
      value=""
    />
  </label>
</div>
`;

exports[`Storyshots InputNumber Controlled InputNumber with initialValue 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputNumber"
  >
    <p
      className="label"
    >
      Number Input Field
       
      <span
        className="required"
      >
        *
      </span>
    </p>
    <input
      className="input"
      name="inputNumber"
      onChange={[Function]}
      placeholder={null}
      type="number"
      value="10"
    />
  </label>
</div>
`;

exports[`Storyshots InputNumber Label 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputNumber"
  >
    <p
      className="label"
    >
      Number Input Field
       
    </p>
    <input
      className="input"
      name="inputNumber"
      onChange={[Function]}
      placeholder={null}
      type="number"
      value="0"
    />
  </label>
</div>
`;

exports[`Storyshots InputNumber Placeholder 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputNumber"
  >
    <p
      className="label"
    >
       
    </p>
    <input
      className="input"
      name="inputNumber"
      onChange={[Function]}
      placeholder="0"
      type="number"
      value=""
    />
  </label>
</div>
`;

exports[`Storyshots InputNumber Required 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputNumber"
  >
    <p
      className="label"
    >
      Number Input Field
       
      <span
        className="required"
      >
        *
      </span>
    </p>
    <input
      className="input"
      name="inputNumber"
      onChange={[Function]}
      placeholder={null}
      type="number"
      value="0"
    />
  </label>
</div>
`;

exports[`Storyshots InputText Basic 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputText"
  >
    <p
      className="label"
    >
       
    </p>
    <input
      className="input"
      name="inputText"
      onChange={[Function]}
      placeholder={null}
      type="text"
      value="A static value"
    />
  </label>
</div>
`;

exports[`Storyshots InputText Controlled InputText 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputText"
  >
    <p
      className="label"
    >
      Text Input
       
      <span
        className="required"
      >
        *
      </span>
    </p>
    <input
      className="input"
      name="inputText"
      onChange={[Function]}
      placeholder="enter text here..."
      type="text"
      value=""
    />
  </label>
</div>
`;

exports[`Storyshots InputText Controlled InputText with initial Value 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputText"
  >
    <p
      className="label"
    >
      Text Input
       
      <span
        className="required"
      >
        *
      </span>
    </p>
    <input
      className="input"
      name="inputText"
      onChange={[Function]}
      placeholder="enter text here..."
      type="text"
      value="This is an initial value. It can be changed."
    />
  </label>
</div>
`;

exports[`Storyshots InputText Label 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputText"
  >
    <p
      className="label"
    >
      Text Input
       
    </p>
    <input
      className="input"
      name="inputText"
      onChange={[Function]}
      placeholder={null}
      type="text"
      value="This field has a label"
    />
  </label>
</div>
`;

exports[`Storyshots InputText Placeholder 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputText"
  >
    <p
      className="label"
    >
       
    </p>
    <input
      className="input"
      name="inputText"
      onChange={[Function]}
      placeholder="this is placeholder text..."
      type="text"
      value=""
    />
  </label>
</div>
`;

exports[`Storyshots InputText Required 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#fff",
      "backgroundImage": "linear-gradient(#eee 2px, transparent 2px),
      linear-gradient(90deg, #eee 2px, transparent 2px),
      linear-gradient(#eee 1px, transparent 1px),
      linear-gradient(90deg, #eee 1px, transparent 1px)",
      "backgroundPosition": "-2px -2px, -2px -2px, -1px -1px, -1px -1px",
      "backgroundSize": "100px 100px, 100px 100px, 10px 10px, 10px 10px",
      "minHeight": "100vh",
      "padding": "40px",
      "width": "100vw",
    }
  }
>
  <label
    className="wrapper"
    htmlFor="inputText"
  >
    <p
      className="label"
    >
      Text Input
       
      <span
        className="required"
      >
        *
      </span>
    </p>
    <input
      className="input"
      name="inputText"
      onChange={[Function]}
      placeholder={null}
      type="text"
      value="This field is required"
    />
  </label>
</div>
`;
